addons/mail/static/src/discuss/call/common/input_device_select.js

```diff
@@ -0,0 +1,47 @@
+import { Component, onWillStart, useState } from "@odoo/owl";
+
+import { _t } from "@web/core/l10n/translation";
+import { browser } from "@web/core/browser/browser";
+import { useService } from "@web/core/utils/hooks";
+
+export class InputDeviceSelect extends Component {
+    static props = ["kind"]; // "videoinput" | "audioinput"
```

Comment:
Instead of comment could use object syntax for props validation, so that can use `validate` function to check value of "kind".
https://github.com/odoo/owl/blob/master/doc/reference/props.md#props-validation